modules/tinymce/src/themes/silver/main/ts/ui/view/ViewWrapper.ts

```diff
@@ -0,0 +1,180 @@
+import {
+  AlloyComponent, AlloyEvents, Behaviour, Composing,
+  Replacing, Sketcher, SlotContainer, SlotContainerTypes, UiSketcher
+} from '@ephox/alloy';
+import { FieldSchema, StructureSchema } from '@ephox/boulder';
+import { View as BridgeView } from '@ephox/bridge';
+import { Arr, Fun, Obj, Optional } from '@ephox/katamari';
+import { Attribute, Css } from '@ephox/sugar';
+
+import { UiFactoryBackstage, UiFactoryBackstageProviders } from '../../backstage/Backstage';
+import { SimpleBehaviours } from '../alien/SimpleBehaviours';
+import View from './View';
+import { ViewConfig } from './ViewTypes';
+
+const makeViews = (parts: SlotContainerTypes.SlotContainerParts, viewConfigs: ViewConfig, providers: UiFactoryBackstageProviders) => {
+  return Obj.mapToArray(viewConfigs, (config, name) => {
+    const internalViewConfig: BridgeView.View = StructureSchema.getOrDie(BridgeView.createView(config));
+
+    return parts.slot(name, View.sketch({
+      dom: {
+        tag: 'div',
+        classes: [ 'tox-view' ]
+      },
+      viewConfig: internalViewConfig,
+      components: [
+        ...internalViewConfig.buttons.length > 0 ? [
+          View.parts.header({
+            buttons: internalViewConfig.buttons,
+            providers
+          })
+        ] : [],
+        View.parts.pane({})
+      ]
+    }));
+  });
+};
+
+const makeSlotContainer = (viewConfigs: ViewConfig, providers: UiFactoryBackstageProviders) => SlotContainer.sketch((parts) => ({
+  dom: {
+    tag: 'div',
+    classes: [ 'tox-view-wrap__slot-container' ]
+  },
+  components: makeViews(parts, viewConfigs, providers),
+  slotBehaviours: SimpleBehaviours.unnamedEvents([
+    AlloyEvents.runOnAttached((slotContainer) => SlotContainer.hideAllSlots(slotContainer))
+  ])
+}));
+
+const getCurrentName = (slotContainer: AlloyComponent) => {
+  return Arr.find(SlotContainer.getSlotNames(slotContainer), (name) =>
+    SlotContainer.isShowing(slotContainer, name)
+  );
+};
+
+const hideContainer = (comp: AlloyComponent) => {
+  const element = comp.element;
+  Css.set(element, 'display', 'none');
+  Attribute.set(element, 'aria-hidden', 'true');
+};
+
+const showContainer = (comp: AlloyComponent) => {
+  const element = comp.element;
+  Css.remove(element, 'display');
+  Attribute.remove(element, 'aria-hidden');
+};
+
+const makeViewInstanceApi = (slot: HTMLElement): BridgeView.ViewInstanceApi => ({
+  getContainer: Fun.constant(slot)
+});
+
+const runOnShow = (slotContainer: AlloyComponent, name: string) => {
+  SlotContainer.getSlot(slotContainer, name).each((view) => {
```

Comment:
It's probably overkill, but you could pull out a `runOnPaneWithInstanceApi` function here.

Reply:
I added a function to not repeat these two things.